### ISSUE TITLE
✨ NEW: Add support for PBS

### ISSUE THREAD

**Issue by mbercx** [aiidateam/aiida-hyperqueue#4]
Currently the scheduler has only been developed/tested with Slurm in mind. However, since HyperQueue also supports PBS, we should redesign some parts to be more general so both job managers can be supported.

**Comment by giovannipizzi** [comment]
Indeed, we should just mirror the options that HQ has already? Maybe we could have some metadata to say that that computer has SLURM and not PBS, but for now I would do it "dumb" and the user has to specify `pbs` or `slurm` as you do in the `hq` command line
